File: src/main/java/com/sabkar/attendance/entity/domain/Bell.java
package com.sabkar.attendance.entity.domain;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;
import java.util.ArrayList;
import java.util.Date;
import java.util.List;

@Entity
@Table(name = "bells")
@Setter
@Getter
public class Bell {
    @Id
    @Column(name = "id", nullable = false)
    private Integer id;
    @Temporal(TemporalType.TIME)
    private Date startTime;
    @Temporal(TemporalType.TIME)
    private Date endTime;

    @OneToMany(mappedBy = "bell", orphanRemoval = true, fetch = FetchType.LAZY)
    private List<Task> tasks = new ArrayList<>();

}

File: fake-api-server/models/Bell.js
const NAME = 'bells';

const generate = () => {
  return [
    {
      id: 1,
      startTime: '08:15:00',
      endTime: '09:35:00',
    },
    {
      id: 2,
      startTime: '09:45:00',
      endTime: '11:05:00',
    },
    {
      id: 3,
      startTime: '11:25:00',
      endTime: '12:45:00',
    },
    {
      id: 4,
      startTime: '12:50:00',
      endTime: '14:10:00',
    },
  ]
}

module.exports = {
  NAME, generate
}